egui/src/containers/frame.rs

```diff
@@ -23,7 +60,7 @@ impl Frame {
     /// For when you want to group a few widgets together within a frame.
     pub fn group(style: &Style) -> Self {
         Self {
-            margin: Vec2::splat(6.0), // symmetric looks best in corners when nesting
+            margin: Vec2::splat(6.0).into(), // symmetric looks best in corners when nesting
```

Comment:
Perhaps we could have a constructor for this too:

```suggestion
            margin: Margin::same(6.0), // symmetric looks best in corners when nesting
```